pom.xml

```diff
@@ -1254,5 +1265,6 @@
         <module>dolphinscheduler-log-server</module>
         <module>dolphinscheduler-tools</module>
         <module>dolphinscheduler-ui</module>
+        <module>dolphinscheduler-scheduler</module>
```

Comment:
Can we rename to the same formats as other plugins? Like `dolphinscheduler-scheduler-plugin-quartz`